pykechain/models/form.py

```diff
@@ -478,3 +477,52 @@ def apply_transition(self, transition: Transition):
                 response=response,
             )
         self.refresh(json=response.json()["results"][0])
+
+    def has_part(self, part: Part) -> bool:
+        """Return boolean if given Part is part of the Form tree.
+
+        Based on the Parts category, either the model or the instance tree is checked.
+
+        :param part: a Part object
+        :raises IllegalArgumentError: in case not a `Part` object is used when calling the method
+        :raises APIError: in case an Error occurs when checking whether the `Form` contains the
+        `Part`
+
+        """
+        part_id = check_base(part, Part)
+
+        url = self._client._build_url("forms_has_part",
+                                      form_id=self.id,
+                                      part_id=part_id)
+        response = self._client._request(
+            "GET", url=url
+        )
+        if response.status_code != requests.codes.ok:
+            raise APIError(
+                "Could not process whether `Form` {} has part {}".format(self.id, part_id),
```

Comment:
please use the f-strings

```suggestion
                f"Could not process whether `Form` {self.id} has part {part_id}",
```